src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerConversionExtensions.cs

```diff
@@ -47,7 +47,7 @@ internal static class JaegerConversionExtensions
 
         private static readonly Dictionary<string, int> PeerServiceKeyResolutionDictionary = new Dictionary<string, int>(StringComparer.OrdinalIgnoreCase)
         {
-            ["peer.service"] = 0, // peer.service primary.
+            [SpanAttributeConstants.PeerServiceKey] = 0, // peer.service primary.
```

Comment:
Please move to separate PR.

Reply:
All I did here was move "peer.service" to a constant so I could check it in the unit test(s).